UPGRADING.rst

```diff
@@ -11,6 +11,37 @@ reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
 
+#4673 Eliminate burner accounts
+===============================
+
+When this PR lands in the dev deployment, in `platform-hca-dev` Google cloud
+project, the operator should perform the following steps, and then announce for
+other developers to do the same.
+
+#) Delete your burner Google service account:
+
+    #) Go to the Google Cloud console, select the `platform-hca-dev` project,
+       and navigate to ``IAM & Admin`` -> ``IAM``
+
+    #) Select your individual burner account. This is the one where your CruzID
+       is followed by  `….ucsc.edu@gmail.com`; it does not include the string
+       "azul"
+
+    #) Click ``DISABLE SERVICE ACCOUNT`` -> ``DISABLE``
+
+    #) Click ``DELETE SERVICE ACCOUNT`` -> ``DELETE``
+
+#) Delete your burner Google account:
+
+    #) Signin to Google using your burner account, in the icon with your
+       burner's name initial (upper right hand of the page), click the
+       ``Manage your Google Account`` button. And navigate to ``Data & Privacy``
+
+    #) At the bottom of the page, in ``More options``, click on the
+       ``Delete your Google Account`` option, complete Google's requisites and
+       terminate your burner account by clicking on ``Delete Account``
+
+
```

Comment:
Burner accounts are not service accounts. They are regular GMail accounts just like our personal developer accounts. For example, my burner is `nadove.ucsc.edu@gmail.com`. I don't know what the procedure is for deleting a GMail account, but this section is incorrect.